client/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
```

Comment:
Look at paz comments about the validations

Reply:
ok, I added it